packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -392,13 +392,15 @@ export const prepareLocalOrchestrationAccountKit = (
          * @type {HostOf<OrchestrationAccountI['getBalance']>}
          */
         getBalance(denomArg) {
-          // FIXME look up real values
-          // UNTIL https://github.com/Agoric/agoric-sdk/issues/9211
           const [brand, denom] =
             typeof denomArg === 'string'
-              ? [/** @type {any} */ (null), denomArg]
+              ? [chainHub.lookupAsset(denomArg).brand, denomArg]
               : [denomArg, 'FIXME'];
```

Comment:
Seems we can simplify since we're only interested in the brand:
```suggestion
          const brand =
            typeof denomArg === 'string'
              ? chainHub.lookupAsset(denomArg).brand
              : denomArg;
```

It seems we should also update the `orchestrationAccountMethods.getBalance` guard:
```diff
- M.call(M.any())
+ M.call(M.or(BrandShape, M.string())
```

Reply:
Tangential - i wonder if we should rename this LOA method to `getPurseBalance`, and reserve `getBalance` for `/cosmos.bank.v1beta1.QueryBalanceRequest`. The former will not be able to get balances for non vbank assets.

Reply:
That seems worth its own issue.

Or at least a chat with @mitdralla .

Reply:
> we're only interested in the brand
`getBalance` returns a DenomAmounts so we need to figure out a denom if one isn't passed in

Reply:
@turadg thanks, I see now